crates/fs/Cargo.toml

```diff
@@ -1,14 +1,14 @@
 [package]
-name                  = "pacquet-fs"
-description           = "Filesystem utility functions used by pacquet"
-version               = "0.0.1"
-publish               = false
-authors.workspace     = true
-edition.workspace     = true
-homepage.workspace    = true
-keywords.workspace    = true
-license.workspace     = true
-repository.workspace  = true
+name                 = "pacquet-fs"
+description          = "Filesystem utility functions used by pacquet"
+version              = "0.0.1"
+publish              = false
+authors.workspace    = true
+edition.workspace    = true
+homepage.workspace   = true
+keywords.workspace   = true
+license.workspace    = true
+repository.workspace = true
 
```

Comment:
I don't really like this alignment with fixed width before =. It means that if there is a new field with a longer length, the all the other unrelated line will be change too, making diffs bigger.

Reply:
@zkochan I tried removing `align_width = true` from `.taplo.toml` before making this PR. The `Cargo.toml` files look messy without column alignment, so I decided to keep it as it is.